tests/bears/BearTest.py

```diff
@@ -179,3 +180,14 @@ def test_new_result(self):
         result = bear.new_result('test message', '/tmp/testy')
         expected = Result.from_values(bear, 'test message', '/tmp/testy')
         self.assertEqual(result, expected)
+
+    def test_download_cached_file(self):
+        url = 'https://google.com'
+        filename = 'google.html'
+        uut = TestBear(self.settings, None)
+        result_filename = uut.download_cached_file(url, filename)
+        expected_filename = join(uut.data_dir, filename)
+        self.assertEqual(result_filename, expected_filename)
+        result_filename = uut.download_cached_file(url, filename)
+        self.assertEqual(result_filename, expected_filename)
```

Comment:
> It is also not checked if the file really exists
trivial implementation. We can check it if you want. I thought it wasn't completely necessary